frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -254,90 +245,31 @@ export default {
         this.$emit("request_file_change", direction, file);
       }
     },
-    save: async function(){
-      this.has_changed = false
-      this.save_loading = true
-      let url;
-      if (this.task && this.task.id) {
-        url = `/api/v1/task/${this.task.id}/annotation/update`;
-      } else {
-        url = `/api/project/${this.project_string_id}/file/${this.file.id}/annotation/update`
-      }
-
-      const res = await postInstanceList(url, this.instance_list.get_all())
-      const { added_instances } = res
-      this.instance_list.get_all().map(instance => {
-        const instance_uuid = instance.creation_ref_id
-        const updated_instance = added_instances.find(added_instance => added_instance.creation_ref_id === instance_uuid)
-        if (updated_instance) {
-            instance.id = updated_instance.id
-        }
-      })
-      this.save_loading = false
-    },
-    trigger_task_change: async function (direction, assign_to_user = false) {
-      if (this.has_changed) {
-        await this.save();
-        await this.save();
-      }
-      this.$emit("request_new_task", direction, this.task, assign_to_user);
-    },
-    on_task_annotation_complete_and_save: async function () {
-      await this.save();
-      await this.save();
-      const response = await finishTaskAnnotation(this.task.id);
-      const new_status = response.data.task.status;
-      this.task.status = new_status;
-      if (new_status !== "complete") {
-        this.submitted_to_review = true;
-      }
-      if (this.$props.task && this.$props.task.id) {
-        this.save_loading_image = false;
-        this.trigger_task_change("next", this.$props.task, true);
-      }
-    },
     update_attribute: function(attribute) {
       const command = new UpdateInstanceAttributeCommand([this.instance_list.get().find(inst => inst.creation_ref_id === this.current_instance.creation_ref_id)], this.instance_list)
       command.set_new_attribute(attribute[0].id, {...attribute[1]})
-      this.command_manager.executeCommand(command)
+      this.annotation_ui_context.command_manager.executeCommand(command)
```

Comment:
Use `command_manager` on `this.annotation_ui_context` because its shared with other components. This logic was copied from `text_annotation_core`.

Moving forward we want `audio_annotatin_core` to fire different events like `create_annotation`, `update_annotation` and have ancestor component catch them and call ` this.annotation_ui_context.command_manager.executeCommand` method. `audio_annotation_core` should not know about existence of the command manager.